src/pages/SocialGraph.vue

```diff
@@ -148,32 +202,50 @@ export default {
           );
           return account;
         });
-      return (await Promise.all(promises)).sort(
-        (a, b) => b.totalValue - a.totalValue,
-      );
```

Comment:
Not sure if we can keep this sorting, this is removed to avoid confusion?

Reply:
hmmm... we can keep it just in a page range, not cross pages.